### PR TITLE
added white brand. Reworked colour scheme

### PR DESCRIPTION
**Pull request by ghost**
**Reworked the UI**
-------

### Why I changed this?
I felt that the black and blue theme was a bit too dull so I decided to add some pop to the ui by converting the logo to white and making blue the primary colour.

**Screenshot of change**: [here](https://imgur.com/6AkYDLr)

### CODE REVIEW

File: src/main.css [hunk 1]
```diff
@@ -8,9 +8,8 @@
 }
 
 html {
-    background: #00162d;
-    box-sizing: border-box;
-    color: #f6f6f6;
+    background: #282c37;  /* fallback for old browsers */
```
Comment (Siilwyn) [inline src/main.css]: Fallback for old browsers?

File: src/main.css [hunk 2]
```diff
@@ -8,9 +8,8 @@
 }
 
 html {
-    background: #00162d;
-    box-sizing: border-box;
```
Comment (Siilwyn) [inline src/main.css]: Why remove this?

File: src/main.css [hunk 3]
```diff
@@ -8,9 +8,8 @@
...
-    color: #f6f6f6;
+    background: #282c37;  /* fallback for old browsers */
+    color:#FFF;
```
Comment (Siilwyn) [inline src/main.css]: Lowercase please: #fff. Also missing space after colon

File: src/partials/app-header/app-header.css [hunk 4]
```diff
@@ -19,11 +20,7 @@
 
 .app-header-logo {
     height: 3rem;
-    transition: filter 0.5s;
-}
-
-.app-header-logo:hover {
-    filter: drop-shadow(0 0 0.3rem #1ca6df);
+    transition: filter 0.2s;
```
Comment (Siilwyn) [inline src/partials/app-header/app-header.css]: without a hover effect the transition is not needed anymore right?

File: src/partials/app-header/app-header.css [hunk 5]
```diff
@@ -34,38 +31,20 @@
     display: flex;
     justify-content: center;
     align-items: center;
+    transition:all 0.2s ease-in-out;
```
Comment (Siilwyn) [inline src/partials/app-header/app-header.css]: missing space

File: src/partials/app-header/app-header.css [hunk 6]
```diff
@@ -34,38 +31,20 @@
...
-.app-header-item:active {
-    background: #1997cc;
-    color: black;
-}
 
 .app-header-item[disabled=true] {
     display: none;
 }
 
-.app-header-item::after {
```
Comment (Siilwyn) [inline src/partials/app-header/app-header.css:53]: Why remove this effect? Now there is no indication of interactivity at all

**Review by Siilwyn — COMMENTED** [review]
Shouldn't commit `package-lock.json`.

File: src/main.css (hunk 1 above)
Comment (ghost) [reply src/main.css]: I was testing gradients and that was a fallback if it wasn't supported. Didn't remove the comment :P

File: src/main.css (hunk 2 above)
Comment (ghost) [reply src/main.css]: I will add the box-sizing back, didn't realise I deleted that. My bad

File: src/main.css (hunk 3 above)
Comment (ghost) [reply src/main.css]: Will do!

**Comment by Siilwyn** [comment]
Whoops might not be clear. I meant don't commit the `package-lock.json` in this PR since it has nothing to do with these changes. Should still keep it, just don't commit it.

**Pull request closed without merging**